app/src/main/java/com/asfoundation/wallet/ui/iab/localpayments/LocalPaymentInteractor.kt

```diff
@@ -39,39 +37,40 @@ class LocalPaymentInteractor(private val deepLinkRepository: InAppDeepLinkReposi
           .flatMap { smsValidationInteract.isValidated(it) }
           .onErrorReturn { true }
 
-  fun getPaymentLink(domain: String, skuId: String?, originalAmount: String?,
-                     originalCurrency: String?, paymentMethod: String, developerAddress: String,
+  fun getPaymentLink(packageName: String, fiatAmount: String?, fiatCurrency: String?,
+                     paymentMethod: String, productName: String?, type: String, origin: String?,
+                     walletDeveloper: String?, developerPayload: String?,
                      callbackUrl: String?, orderReference: String?,
-                     payload: String?, referrerUrl: String?): Single<String> {
-
+                     referrerUrl: String?): Single<String> {
     return walletService.getAndSignCurrentWalletAddress()
         .flatMap { walletAddressModel ->
           Single.zip(
-              partnerAddressService.getStoreAddressForPackage(domain),
-              partnerAddressService.getOemAddressForPackage(domain),
+              partnerAddressService.getStoreAddressForPackage(packageName),
+              partnerAddressService.getOemAddressForPackage(packageName),
               BiFunction { storeAddress: String, oemAddress: String ->
-                DeepLinkInformation(storeAddress, oemAddress)
+                Pair(storeAddress, oemAddress)
```

Comment:
Wouldn't be more readable to leave DeepLinkInformation data class instead of Pair?